smart_control/notebooks/SAC_Demo.ipynb

```diff
@@ -151,10 +151,10 @@
         "  logging.info(*args)\n",
         "  print(*args)\n",
         "\n",
-        "data_path = \"third_party/py/smart_buildings/smart_control/configs/resources/sb1/\" #@param {type:\"string\"}\n",
-        "metrics_path = \"/usr/local/google/home/metrics\" #@param {type:\"string\"}\n",
-        "output_data_path = '/usr/local/google/home/smart_buildings/smart_control/sb_colab_demo' #@param {type:\"string\"}\n",
-        "root_dir = \"/usr/local/google/root\" #@param {type:\"string\"}\n",
+        "data_path = \"/home/gabriel-user/projects/sbsim-2/sbsim/smart_control/configs/resources/sb1/\" #@param {type:\"string\"}\n",
```

Comment:
I am going to implement environment variables to help us customize these paths without updating the source code. #23

Reply:
For now we can keep these diffs. I want to prioritize merging this PR first and then I will implement environment variables afterwards.